### ISSUE TITLE
[request] The need of stock asset registration skip

### ISSUE THREAD

**Issue by upl1nk-v01d** [callofduty4x/CoD4x_Server#400]
For adding own customized assets (weapons, models etc.) to mod.ff, the modification in server's source code, e.g. new codeblock implementation in function, that is responsible registering xassets in memory, for skipping rarely used stock mp assets (for example flagged weapon asset names in server gsc script or listed asset names in txt file in cod4x mod dir like "binoculars_mp", "aw_50_mp" or even used items like "ak47_silencer_mp") is needed because of cod4x server xassets count limitations. The stock ff file's and iwd file's assets are automatically registered and loaded into the memory and no prevention of this is available. By doing an implementation the result would be freeing necessary g_poolSize slots and therefore decreasing registered asset count to asset types like ASSET_TYPE_WEAPON.
Otherwise by adding too many mod assets results in max assets excession error and server crash. The workaround for now is the stock assets like weapons in mod\weapons\mp replacement with customized ones.

The bottom line: please make available an option to skip automatic registering and loading stock mp xassets.

P.S.
128 bone error is caused by weapon stowing. 
This error can be solved by commenting out the following lines in _weapons.gsc:
stow_on_back();
stow_on_hip();

**Comment by IceNinjaman** [comment]
You can override almost all stock fast-files with a mod. Mods can provide their own common_mp.ff, for example.

**Comment by proxict** [comment]
Closing due to inactivity.
